room_cancelation/auto_cancel_event.py

```diff
@@ -0,0 +1,57 @@
+from api.room.models import Room as RoomModel
+from api.events.models import Events as EventsModel
+from datetime import datetime
+from helpers.calendar.credentials import Credentials
+from dateutil.relativedelta import relativedelta
+from helpers.calendar.events import RoomSchedules
+
+
+class UpdateRecuringEvent():
+    def update(self):
+        query = RoomModel.query
+        now = datetime.utcnow().isoformat()+"Z"
+        next_day = (datetime.utcnow() + relativedelta(day=1)).isoformat()+"Z"
+        events = RoomSchedules().get_all_recurring_events(
+            query, now, next_day)
+        for event in events:
+            start_date = event["start_date"]
+            end_date = event["end_date"]
+            new_recurring_event = EventsModel(
+                event_id=event["recurring_event_id"],
+                room_id=event["room_id"],
+                event_title=event["event_summary"],
+                start_time=start_date,
+                end_time=end_date,
+                checked_in=False,
+                cancelled=False)
+            event_query = EventsModel.query
+            calendar_id = event["calendar_id"]
+            event_id = event["recurring_event_id"]
+            service = Credentials().set_api_credentials()
+            missed_checkins = event_query.filter(
+                EventsModel.event_id == event["recurring_event_id"] and
+                EventsModel.checked_in == "False")
+            missed_checkins_count = missed_checkins.count()
```

Comment:
Is there a need of having this variable here instead of just calling missed_checkins.count() directly?

Reply:
We are updating it ASAP

Reply:
Fixed @Bonifase

Reply:
@Borniface we have updated it.